tensorflow/python/lib/io/tf_record.py

```diff
@@ -69,7 +69,7 @@ def __init__(self,
     Leaving an option as `None` allows C++ to set a reasonable default.
 
     Args:
-      compression_type: `TFRecordCompressionType` or `None`.
+      compression_type: `"GZIP"`, "ZLIB" or `"NONE"`.
```

Comment:
Are these docs correct? It looks like only `0`, `1`, and `2` are valid
values:

```
  File "/usr/local/google/home/wchargin/virtualenv/tf-nightly-2.0-preview-20190321-py2.7/local/lib/python2.7/site-packages/tensorflow/python/lib/io/tf_record.py", line 126, in get_compression_type_string
    raise ValueError('Not a valid compression_type: "{}"'.format(options))
ValueError: Not a valid compression_type: "NONE"
```

It seems pretty unergonomic to require literal `0`, `1`, and `2` as
integer arguments. Having an enum, as prior to this commit, is clearly
the nicest invocation, but if we can’t have that for whatever reason
then can we at least allow the user to pass strings rather than
integers? that is, can we change the implementation to match the
specification rather than the other way around?

Reply:
Amendment: `"GZIP"` and `"ZLIB"` work, but `"NONE"` does not; the magic
string for “none” appears to be`""`, which is presumably distinct from
`None` meaning “default value” (ugh). So merely a minor docs fix
required.

Reply:
@yongtang is addressing this in #27041; thanks!